float32 goal_x
float32 goal_y
---
duration elapsed_time
---
duration elapsed_time
